free-programming-books.md

```diff
@@ -1068,8 +1069,10 @@ Kerridge (PDF) (email address *requested*, not required)
 * [Programming in C](https://www.freetechbooks.com/programming-in-c-t1337.html) - Kishori Mundargi
 * [Structures and C](https://www.smashwords.com/books/view/644937) - Prakash Hegade
 * [The C book](http://publications.gbdirect.co.uk/c_book/) - Mike Banahan, Declan Brady and Mark Doran (PDF, HTML)
+* [The C Programming Language](https://inspirit.net.in/pdfviewer/web/viewer.html?file=..%2F..%2F.%2Fbooks%2Fc%20plus%20plus%2FThe%20C%20Programming%20Language%20-%20Brian%20Kernighan%20%26%20Dennis%20Ritchie.pdf) - Dennis M. Ritchie, Brian W. Kernighan (PDF)
```

Comment:
Not free. I asked Dennis about the book; he said he was working on a way to allow authorized free downloads, but AFAIK, it hasn't happened yet. Please remove this.